netplan/cli/commands/try_command.py

```diff
@@ -47,13 +47,26 @@ def __init__(self):
         self._config_manager = None
         self.t_settings = None
         self.t = None
+        self._rootdir = os.environ.get('DBUS_TEST_NETPLAN_ROOT', '/')
+        self._netplan_try_stamp = os.path.join(self._rootdir, 'run', 'netplan', 'netplan-try.ready')
 
     @property
     def config_manager(self):  # pragma: nocover (called by later commands)
         if not self._config_manager:
             self._config_manager = ConfigManager()
         return self._config_manager
 
+    def clear_ready_stamp(self):
+        if os.path.isfile(self._netplan_try_stamp):
+            os.remove(self._netplan_try_stamp)
+            return True
+        return False
+
+    def touch_ready_stamp(self):
+        os.makedirs(self._rootdir + '/run/netplan', mode=0o700, exist_ok=True)
+        open(self._netplan_try_stamp, 'w').close()
+        return True
```

Comment:
(nitpick) Given that this never returns False I wonder if we need a return value here ? I see it's used in the tests but I think it's okay to just not have a return value here, the side-effect of touching a file is something that can be observed.

Reply:
ACK.